draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -406,6 +406,36 @@ On the server side, this mostly helps avoid resource exhaustion. For clients, pr
 
 The value of the idle timeout to use depends on the exact deployment and is a trade-of between resource usage on clients/servers and the overhead of opening new connections. Very short timeouts that are at or below the timeouts used for application layer watchdogs, typically in the range of 30-60s can be considered unreasonable. In contrast, the upper limit is much more difficult to define but may be in the range of 10 to 15min, depending on the available resources, or never (disabling idle timeout) in scenarios where a permanently open connection is required.
 
+## Malformed Packets and Unknown clients
+
+The RADIUS specifications say that an implementation should "silently discard" a packet in a number of circumstances.
+This action has no further consequences for UDP based transports, as the "next" packet is completely independent of the previous one.
+
+When TLS is used as transport, decoding the "next" packet on a connection depends on the proper decoding of the previous packet.
+As a result the behavior with respect to discarded packets has to change. With DTLS, while the "next" packet does not depend on proper decoding of the previous packet, but since DTLS provides
+integrity protection, any decoding error should be treated as a broken connection.
+
+Implementations of this specification SHOULD tread the "silently discard" texts in the RADIUS specification referenced above as "silently discard and close the connection".
+That is, the implementation SHOULD send a TLS close notification and the underlying connection MUST be closed if any of the following circumstances are seen:
```

Comment:
```suggestion
That is, the implementation SHOULD send a TLS close notification and, in the case of RADIUS/TLS, the underlying TCP connection MUST be closed if any of the following circumstances are seen:
```

The original text was specific for RADIUS/TLS, and RADIUS/DTLS does not have a "connection", and closing the UDP socket is an implementation detail.
I don't have a strong opinion about it, but that's what just came to my mind.